### DISCUSSION TITLE
Monospace font in Markdown Visual editor

### DISCUSSION THREAD

**Discussion by maiertech** [standardnotes/forum#2581] [Suggestions]
**Describe the bug**

Fenced code blocks and the page source code are rendered with sans serif font. They should be rendered with a monospaced font, which is more appropriate for source code.

**To Reproduce**
- Create a fenced codeblock and look at how it is rendered.
- Switch to page source code view.

**Expected behavior**

Rendering with monospaced font to preserve code formatting and for better legibility.

**Screenshots**

Fenced code block with sans serif font:

![fenced-code-block](https://user-images.githubusercontent.com/1482402/169407383-c065e575-5ae1-4d06-b549-569f0150a2e1.png)

Page source code with sans-serif font:

![source-code-view](https://user-images.githubusercontent.com/1482402/169407660-5b7974ab-0779-43da-9879-814172bc731c.png)

**Desktop (please complete the following information):**
 - OS: macOS 12.4
 - Version: 3.20.0

**Comment by lernisto** [comment]
*Some* types of code look better with a monospace font. CSS isn't one of them, IMO.

The editor *should* let you choose whatever font you want, however.

**Comment by maiertech** [comment]
> _Some_ types of code look better with a monospace font. CSS isn't one of them, IMO.
> 
> The editor _should_ let you choose whatever font you want, however.

Code needs indentation, which only works with monospace fonts.

**Reply by terrelcodes** [reply]
All spaces are the same width whether the font is monospace or not, so indentation (leading whitespace) is always consistent.
